TrafficCapture/coreUtilities/build.gradle

```diff
@@ -18,65 +18,44 @@ buildscript {
 
 plugins {
     id 'org.opensearch.migrations.java-library-conventions'
-//    id "com.github.spotbugs" version "4.7.3"
-//    id 'checkstyle'
-    id "io.freefair.lombok" version "8.0.1"
+    id 'io.freefair.lombok'
     id 'java'
     id 'java-test-fixtures'
 }
 
-//spotbugs {
-//    includeFilter = new File(rootDir, 'config/spotbugs/spotbugs-include.xml')
-//}
-
-//checkstyle {
-//    toolVersion = '10.12.3'
-//    configFile = new File(rootDir, 'config/checkstyle/checkstyle.xml')
-//    System.setProperty('checkstyle.cache.file', String.format('%s/%s',
-//            buildDir, 'checkstyle.cachefile'))
-//}
-
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    api platform("io.opentelemetry:opentelemetry-bom:1.34.1")
+    api project(":trafficCaptureDependencies")
```

Comment:
why do these need to be defined as API dependencies?  Since the actual dependencies in the dependencies project are empty, what would different dependency types mean?

Reply:
Never got a good answer, it seems like `java` projects didn't support API imports.  While it would be nice to know I'm not sure this matters. 🤷